add SDC file type to radiant

johnathan-convertino-afrl opened the pull request:
SDC files are supported and allow for multiple constraint files vs only one with PDC.

olofk commented:
Thanks. Could you also update the reference files accordingly so that the test suite passes? Some more info about testing and using the `GOLDEN_RUN` env var to update files automatically can be found here https://edalize.readthedocs.io/en/latest/dev/tests.html#testcases

johnathan-convertino-afrl commented:
Neat, I'll try to get on this soon(tm).

johnathan-convertino-afrl commented:
Ok a few issues, added it, tried it and it didn't seem to work. Could you give me a few pointers here? I assumed this should of been the exact same as the add for systemVerilog, but that fails.

Also Python 3.9 and greater are broken due to a issue unrelated to mine. Looks like a numpy version error.

johnathan-convertino-afrl commented:
Ok, so I've fixed the issue on my end. Though it looks like python 3.9 and up is broken due to a numpy version conflict. Nothing to do with this commit so I'm leaving it broken.

Radiant SDC test passes and fixed.

olofk commented:
Alright. Picked and pushed! Thank you for your contributions.
